README.md

```diff
@@ -88,6 +89,7 @@ jobs:
 - `namespace`: Namespace of the nomad job
 - `job`: Name of the nomad job
 - `parameterized`: Set to true if the nomad job is parameterized
+- `allow-failure`: Set to true so that the action completes successfully if logs are detected on stderr
```

Comment:
The name of this parameter is very confusing. Something like `fail-on-stderr` or `allow-stderr` is more correct, and a lot less confusing.

Reply:
agreed, I renamed the input to allow-stderr